src/utils/theme.js

```diff
@@ -54,9 +59,10 @@ export default {
   colors: {
     white: '#fff',
     text: '#333333',
-    outline: '#6be9c1',
-    secondaryText: '#757575',
     textLight: '#828282',
+    secondaryText: '#757575',
+    outline: '#6be9c1',
+    vibrant: '#6be9c1',
```

Comment:
indeed :)
I'm yet to make a PR to improve this color variables.
So, we have this #6be9c1, which is exported as `outline` but in fact it can be used in things other than an outline - such the bg on the dropdown - so I think it's better to have `vibrant` color variable, since it's more generic than just `outline`.

IMO, The correct way to handle these colours would be as we've discussed some weeks ago:

```js
const vibrant = '#6be9c1'
....

export const colors = {
  outline: vibrant,
  vibrant
}
```

But I think this is out of the scope of this PR

Reply:
I dunno. I think that, if introducing a new variable is within this PR, then the way that you introduce it is also within scope. No?

Because there's lots of ways you could do this. You could
- create a second variable name
- just rename `outline` to `vibrant`, because it's no longer used just for outlines
- take the approach you've just suggested

And possibly others.

But I reckon that introducing a second name for the same value is the least favourable option. Otherwise, in future PRs, before you've done what you suggest above, other engineers could start using `vibrant` or `outline` interchangeably & creating a bit of a mess :thinking: